lib/cartopy/tests/mpl/test_mpl_integration.py

```diff
@@ -578,6 +633,8 @@ def test_pcolormesh_diagonal_wrap():
     assert hasattr(mesh, "_wrapped_collection_fix")
 
 
+@pytest.mark.skipif(MPL_VERSION.release[:2] >= (3, 8),
+                    reason='redundant from mpl v3.8')
```

Comment:
Have I understood this right?  I think this test relates to the need to initially call `pcolor` with an array of zeros, which will no longer be necessary with the new `QuadMesh` array handling.

Reply:
I think that the number of paths should still be short in the new version, so I _think_ this should still be tested. i.e. we only want to draw the small number of cells, we don't want to draw all pcolor cells and make them invisible.